torch/csrc/jit/tensorexpr/mem_dependency_checker.cpp

```diff
@@ -204,7 +204,7 @@ void AccessInfo::dumpDOT(std::ostream& os) const {
     os << "n" << id_ << " [\n";
     os << "label = \"" << AccessToString(type_) << " (#" << id_ << ")\\n";
     os << "buf : " << *var_ << "\\n";
-    os << "bounds : \[";
+    os << "bounds : [";
```

Comment:
+1 though it perhaps can be submitted as super trivial one-liner.

Reply:
This is triggered by one MSVC warning